### ISSUE TITLE
Accessibillity-Unlabeled image

### ISSUE THREAD

**Issue by timo54** [rokwire/illinois-app#15]
## Android 2.2.11

**Description**
When swiping through the app, the initial focus as displayed in the screen capture below is announced as an "Unlabeled image." (See red box in screen capture.) I don't know what image this is referring to. 

**To Reproduce**
1. Go to Events/Academic evening
2. With VoiceOver on, begin swiping through the screen. 
3. When the list of Available times is reached, 
4. "Unlabeled image" will be announced.

**Expected behavior**
There's no image encountered at this point so there shouldn't be an announcement of an unlabeled image. If there is a hidden image, it should be removed. 

**Screenshots**
See the screen capture below

**Desktop (please complete the following information):**
 - OS: Android
 - Version - 2.2.11

**Smartphone (please complete the following information):**
 - Device: Google Pixel 3
 - OS: Android
<img width="431" alt="UnlabeledImage-mrkup" src="https://user-images.githubusercontent.com/12497650/91495527-489e1900-e880-11ea-9dc2-3208fc1cdb57.png">

**Comment by TodorBachvarov** [comment]
Fixed

**Comment by acovher2** [comment]
Verified as fixed

**Comment by acovher2** [comment]
Verified in version 2.5.9 on Android 10 and iOs 14.4

**Comment by acovher2** [comment]
Closed wrong issue, apologies

**Comment by acovher2** [comment]
Verified as resolved in 2.6.28 9/30
